### ISSUE TITLE
'High Contrast Black' theme Tree same text and background color 

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1021]
**Describe the bug**
The Tree component has the same color for text and background - white. So table contents are not readable.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/tree.html
2. Check the Tree component

**Expected behavior**
The tree component should have a black background color for HCB theme.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81760770-fa097200-947c-11ea-8cb8-8d3344336777.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 12.2:1
<img width="897" alt="Screen Shot 2020-10-07 at 5 48 40 PM" src="https://user-images.githubusercontent.com/21978807/95529976-64253500-0991-11eb-9fee-df2bdf16430e.png">
